zerver/views/auth.py

```diff
@@ -254,13 +254,15 @@ def finish_desktop_flow(request: HttpRequest, user_profile: UserProfile,
             'subdomain': realm.subdomain}
     token = store_login_data(data)
 
-    return create_response_for_otp_flow(token, otp, user_profile)
+    return create_response_for_otp_flow(token, otp, user_profile,
+                                        encrypted_key_field_name='otp_encrypted_one_time_login_key')
```

Comment:
I think `otp_encrypted_login_key` is probably a better name for this?  I'm not sure the variable name needs to emphasize how reusable it is.

Reply:
Ah, that's much better. Naming things is hard :confounded: